apiVersion: openapi/v1alpha1
kind: Radar
metadata:
    name: "25418688"
    labels:
        datastore_id: "5035131190378496"
data:
    classification: Serious Bug
    created: "2016-03-29T18:31:19.32179Z"
    description: "Summary:\r\nWhile testing a Mac App Store app, and entering in my sandbox account’s Apple ID credentials, I get prompted with “This Apple ID has not yet been used with the App Store” and need to fill in a bunch of information. The only workaround I could find (described in this Stack Overflow answer: http://stackoverflow.com/a/34716634/105717) involves killing a few daemons (storeaccountd, storeassetd, storedownloadd) and removing their caches in ~/Library/Caches. That fixes the issue at hand, allowing me to authenticate, then run the app\r\n\r\nSteps to Reproduce:\r\n1. Restart Mac\r\n2. Launch fresh MAS-ready app signed with the 3rd Party Developer certificate, without an _MASReceipt folder\r\n3. When prompted for Apple ID credentials, enter a sandbox testing account username and password\r\n\r\nExpected Results:\r\nThe credentials are authenticated and an _MASReceipt folder gets generated\r\n\r\nActual Results:\r\nI get the prompt “This Apple ID has not yet been used with the App Store”, and to continue I have to associate a credit card with the TESTING account\r\n\r\nRegression:\r\nThis didn’t happen in 10.10, started only with 10.11\r\n\r\nNotes:\r\nProvide additional information, such as references to related problems, workarounds and relevant attachments."
    email: thedov@gmail.com
    modified: "2016-04-03T16:33:57.02294Z"
    number: "25418688"
    number_intvalue: 25418688
    originated: 29-Mar-2016 02:31 PM
    parent_number: '&{NULL_VALUE}'
    product: OS X
    product_version: ""
    reproducible: Always
    resolved: Duplicate of 24848770
    status: Closed
    title: '"This Apple ID has not yet been used with the App Store" prompt for sandbox testing account'
